### PR TITLE
Alignment fixes and Improvements to license chart

### PR DESCRIPTION
**Pull request by jyasveer**
- Added the responsive classes to the overview cards
- License chart improvement
   * Show number of licenses.
   * Limit the legends in the chart to 4 and show the remaining as 'Others'

### CODE REVIEW

File: src/app/stack/overview/chart-component.ts [hunk 1]
```diff
@@ -25,7 +25,16 @@ import * as c3 from 'c3';
     template: ``,
     encapsulation: ViewEncapsulation.None,
     inputs: ['data', 'chartOptions', 'configs'],
-    styles: [`.ng2-c3{ display:block;}`] // This is required for proper positioning of tooltip
+    styles: [
+        `.ng2-c3 {
+            display:block; // This is required for proper positioning of tooltip
+        }
+        f8-chart .c3-chart-arcs-title {
```
Comment (arunkumars08) [inline src/app/stack/overview/chart-component.ts]: This need not be generic I guess. This can be handled in the individual component level.

**Comment by arunkumars08** [comment]
@jyasveer Did you create a new branch from 'fix-issues' or 'master'? Because this PR doesn't seem to have earlier changes.

File: src/app/stack/overview/chart-component.ts (hunk 1 above)
Comment (arunkumars08) [inline src/app/stack/overview/chart-component.ts]: I tried locally, it works when you provide styling to individual component. You just have to add an encapsulation in the angular decorator.

Comment (jyasveer) [inline src/app/stack/overview/chart-component.ts]: got it. i will change and push the new changes. thanks

**Pull request merged**